src/main/kotlin/dev/stalla/dom/DomParsingExtensions.kt

```diff
@@ -79,25 +81,17 @@ internal fun Node.parseAsTemporalAccessor(): TemporalAccessor? = DateParser.pars
  * @return The [imageBuilder] populated with the DOM node contents.
  */
 @InternalApi
+@Suppress("ComplexMethod")
 internal fun Node.toRssImageBuilder(imageBuilder: RssImageBuilder, namespace: FeedNamespace? = null): RssImageBuilder {
     for (node in childNodes.asListOfNodes()) {
         if (!namespace.matches(node.namespaceURI)) continue
 
         when (node.localName) {
             "description" -> imageBuilder.description(node.textOrNull())
             "height" -> imageBuilder.height(node.parseAsInt())
-            "link" -> {
-                val link = node.textOrNull() ?: continue
-                imageBuilder.link(link)
-            }
-            "title" -> {
-                val title = node.textOrNull() ?: continue
-                imageBuilder.title(title)
-            }
-            "url" -> {
-                val url = node.textOrNull() ?: continue
-                imageBuilder.url(url)
-            }
+            "link" -> node.textOrNull()?.let(imageBuilder::link)
```

Comment:
Getting rid of too many jumps by using `?.let` for further optional assignments.